admin.adoc

```diff
@@ -861,15 +861,12 @@ the following command line.
 grr_config_updater upload_python --file=myfile.py --platform=windows
 ------------------------------------------------------------------------
 
-At the end of this you should see something like:
-------------------------------------------------------------------------
-Uploaded successfully to aff4:/config/python_hacks/myfile.py
```

Comment:
This message is no longer shown (and I couldn't find "Uploaded successfully" in HEAD).

There is a message regarding the config file that was used but that didn't seem particularly useful in the context of this documentation.